src/Main.java

```diff
@@ -43,7 +46,9 @@ public static void main(String[] args) {
         } else if (choice2 > 2) {
             System.out.println("Ви нічого не обрали");
             System.out.println("Ваш стан здоров'я та удачі не змінився ");
-
+        } else if (choice2 <= 0) {
```

Comment:
You have covered values: 1, 2, >2
The rest can be handled just by pure `else` statement, although this way also works